packages/renderer/src/PreferencesNavigation.svelte

```diff
@@ -130,12 +132,13 @@ $: addSectionHiddenClass = (section: string): string => (sectionExpanded[section
       </a>
       <section class="pf-c-nav__subnav {addSectionHiddenClass('extensionsCatalog')}">
         <ul class="pf-c-nav__list">
-          {#each extensions as extension}
+          {#each sortedExtensions as extension}
             <li class="pf-c-nav__item {addCurrentClass(`/preferences/extension/${extension.name}`)}">
               <a
                 href="/preferences/extension/{extension.name}"
                 id="configuration-section-extensions-catalog-{extension.name.toLowerCase()}"
-                class="pf-c-nav__link">{extension.displayName}</a>
+                class="pf-c-nav__link"
+                style="font-weight: 200">{extension.displayName}</a>
```

Comment:
curious why you didn't use font-extralight in the class ?

Reply:
pf-c-nav__link is also setting the font weight, so that doesn't work. I assume that's why the preferences section lower was using style, and matched code.